### PR TITLE
Use python script for generating list.eval and list.train.

### PR DESCRIPTION
**Pull request by zdenop**
This decreases dependency on `bc` program and solves the problem with EOL on Windows.

### CODE REVIEW

File: generate_eval_train.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    split_point = int(ratio * len(lines))
+    output_dir = input_file.resolve().parent
+    train_list = pathlib.Path(output_dir, 'list.train')
+    eval_list = pathlib.Path(output_dir, 'list.eval')
+
+    with open(train_list, 'w', newline='\n') as f1, open(
+        eval_list, 'w', newline='\n'
+    ) as f2:
+        f1.writelines(lines[:split_point])
+        f2.writelines(lines[split_point:])
```
Comment (stweil) [inline generate_eval_train.py:28]:
```suggestion
    with open(train_list, 'w', newline='\n') as f:
        f.writelines(lines[:split_point])
    with open(eval_list, 'w', newline='\n') as f:
        f.writelines(lines[split_point:])
```

Comment (stweil) [reply generate_eval_train.py:28]: I think it looks simpler like that. Is there a reason to implement it differently?

**Review by stweil — CHANGES_REQUESTED** [review]

File: generate_eval_train.py (hunk 1 above)
Comment (zdenop) [reply generate_eval_train.py:28]: As far as I see modern examples and tutorials on the internet: if you open more files with `with` they are always combined into one as in my PR.
Also, python linters do not like single-letter (`f`) variables and for readability, I suggest using different names for eval and train files.

**Comment by stweil** [comment]
Thank you.

**Pull request merged**